scripts/build.sh

```diff
@@ -13,6 +13,9 @@ export PYTHONPATH=$PATHONPATH:./src
 
 python -m azure.cli -h
 
+# Ensure tokens are erased from VCR recordings
+python -m automation.tests.check_vcr_recordings
```

Comment:
One test is removed from the core test bed which was used to ensure tokens are not recorded on the cassettes. Move it to the automation scripts.